### ISSUE TITLE
`no-unsupported-features/node-builtins` does not check for `fs/promises`

### ISSUE THREAD

**Issue by jordanbtucker** [mysticatea/eslint-plugin-node#243]
The original way to import the `fs` Promises API was via one of these options:

```js
const fsp = require('fs').promises
const {readdir} = require('fs').promises
const {promises: fsp} = require('fs')
const {promises: {readdir}} = require('fs')
import {promises as fsp} from 'fs'
// No way to import `readdir` directly via ES modules.
```

In Node.js v14, they added the ability to import the `fs` Promises API via `fs/promises` so you can do things like:

```js
const fsp = require('fs/promises')
const {readdir} = require('fs/promises')
import fsp from 'fs/promises'
import {readdir} from 'fs/promises'
```

`eslint-plugin-node` doesn't seem to check to make sure that `fs/promises` is available in the supported versions of Node.js listed in `package.json`.

**Comment by thernstig** [comment]
`'fs/promises'` is also actually the first method list since Node.js 15 and forward, so it should get larger adoption as time goes, see https://nodejs.org/api/fs.html#fs_promises_api

https://github.com/mysticatea/eslint-plugin-node#readme should then probably get an update for `node/prefer-promises/fs` to also say `enforce require("fs/promises"). Or is that also not supported and should be a separate issue?

**Comment by jordanbtucker** [comment]
@thernstig, that is a good idea, and it should probably be a separate issue. 👍

**Comment by thernstig** [comment]
> @thernstig, that is a good idea, and it should probably be a separate issue. 👍

Done: https://github.com/mysticatea/eslint-plugin-node/issues/260

**Comment by sosukesuzuki** [comment]
Maybe this is fixed in https://github.com/mysticatea/eslint-plugin-node/pull/261
